src/Umbraco.Web.BackOffice/Filters/FilterAllowedOutgoingMediaAttribute.cs

```diff
@@ -2,57 +2,65 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
-using System.Net.Http;
-using System.Web.Http.Filters;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
 using Umbraco.Core;
 using Umbraco.Core.Composing;
 using Umbraco.Core.Models;
 using Umbraco.Core.Models.Membership;
 using Umbraco.Core.Security;
-using Current = Umbraco.Web.Composing.Current;
+using Umbraco.Core.Services;
+using Umbraco.Web.Security;
+
 
 namespace Umbraco.Web.WebApi.Filters
 {
     /// <summary>
     /// This inspects the result of the action that returns a collection of content and removes
     /// any item that the current user doesn't have access to
     /// </summary>
-    internal class FilterAllowedOutgoingMediaAttribute : ActionFilterAttribute
+    internal class FilterAllowedOutgoingMediaAttribute : TypeFilterAttribute
```

Comment:
I thought I saw in the media controller that these attributes were commented out with a note saying to re-enable them when this is transfered to netcore?